pkg/manifests/nginx_test.go

```diff
@@ -40,6 +40,7 @@ var (
 			Annotations: map[string]string{
 				"service.beta.kubernetes.io/azure-load-balancer-internal": "true",
 			},
+			LoadBalancerSourceRanges: []string{"127.1.000.1"},
```

Comment:
can we add a case where this isn't specified to ensure we're testing default behavior?

Reply:
done